### ISSUE TITLE
The model cannot be trained

### ISSUE THREAD

**Issue by Tidus1991** [albertopolito/CarSNN#1]
Hello,

When I run the training code(100x100 no ceil) with default arguments, the loss keeps going to 100, and the outputs always are [0, 0].

Any idea why?
Thank you in advance.

**Comment by albertopolito** [comment]
Hi,

sorry for the delay. Thanks for your comment. You are right, the networks cannot be trained. We forget to specify that before the train of the networks we performed a pre-train step. In these step we do the train of the same networks and adjust weights and biases. Then, in the "real train", we eliminate the biases in order to reduce the use of memory and initialize the weights with the same find before.  

We add the weights found after the pre-train step as we write in "readme" file. Now if you run the code without any changes the weights are initialized with these values.  

However in our paper we do not use the networks "100x100 no ceil" and "50x50 no ceil". Therefore if you find lower or higher accuracies it is normal.

Thanks a lot for your comment it is very helpful. 

Best regards.

Alberto Viale.

**Comment by Tidus1991** [comment]
Viale,

Thank you for your response, it's really helpful.
